tests/expectations/tests/layout.rs

```diff
@@ -38,7 +38,7 @@ impl <T> ::std::clone::Clone for __IncompleteArrayField<T> {
 }
 impl <T> ::std::marker::Copy for __IncompleteArrayField<T> { }
 #[repr(C, packed)]
-#[derive(Debug, Default, Copy)]
```

Comment:
This will disallow previous allowed deriving of `Copy` though if it contains an incomplete array.